vpn/server_config.txt

```diff
@@ -26,6 +26,7 @@ duplicate-cn
 keepalive 10 120
 tls-auth /etc/openvpn/easy-rsa/keys/ta.key 0
 crl-verify /etc/openvpn/crl.pem
+client-config-dir /etc/openvpn/client_conf
```

Comment:
Use this option to enable profile-wise client configs, e.g., compression options...